openupgrade_scripts/scripts/auth_totp/15.0.1.0/post-migration.py

```diff
@@ -0,0 +1,15 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # This method will copy records has scope = '2fa_trusted_device'
+    # from res_users_apikeys to auth_totp_device
+    openupgrade.logged_query(
+        env.cr,
+        """INSERT INTO auth_totp_device (id, name, user_id, scope, index, key)
+            SELECT id, name, user_id, '%s', index, key
+            FROM res_users_apikeys
+            WHERE scope = '2fa_trusted_device'"""
+        % ("browser"),
```

Comment:
@daonamutc1 như này có được không anh?
```suggestion
        """INSERT INTO auth_totp_device (id, name, user_id, scope, index, key)
            SELECT id, name, user_id, 'browser', index, key
            FROM res_users_apikeys
            WHERE scope = '2fa_trusted_device'"""
```

Reply:
@daonamutc1 đồng thời, xóa luôn các bản ghi có `scope = '2fa_trusted_device'` bên `res_users_apikeys` đi chứ nhỉ?